doc/source/architecture/dynamo.rst

```diff
@@ -74,6 +74,26 @@ nodes in each rack, the data load on the smallest rack may be much higher.  Simi
 into a new rack, it will be considered a replica for the entire ring.  For this reason, many operators choose to
 configure all nodes on a single "rack".
 
+.. _transient-replication:
+
+Transient Replication
+~~~~~~~~~~~~~~~~~~~~~
+
+Transient replication allows you to configure a subset of replicas to only replicate data that hasn't been incrementally
+repaired. This allows you to trade data redundancy for storage usage, and increased read and write throughput. For instance,
+if you have a replication factor of 3, with 1 transient replica, 2 replicas will replicate all data for a given token
```

Comment:
Use the 3 replicas upgraded to 5 case. It's not a tradeoff of redundancy at all. We can't message that at all and it's not true.

Reply:
fixed